trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class Trimmer < Decorator
+  def correct_name
+    @component.correct_name.length > 10 ? @component.correct_name[0..10] : @component.correct_name
```

Comment:
- [ ] Your `correct_name` method here returns here 11 characters, instead of 10. Kindly fix that.

  **SUGGESTION:**  Any of the lines below should work:
  ```
  @component.correct_name.length > 10 ? @component.correct_name[0...10] : @component.correct_name
  ```
  or
  ```
  @component.correct_name.length > 10 ? @component.correct_name[0..9] : @component.correct_name
  ```
  ![pola-num](https://user-images.githubusercontent.com/36542986/159521057-19057de4-2ff6-43fb-b5bd-83d77b666c65.jpg)

Reply:
This issue is well resolved! ✔️